common/buildings/wc_common_buildings.txt

```diff
@@ -0,0 +1,749 @@
+﻿### aviary complex
+
+aviary_complex_01 = {
+	construction_time = quick_construction_time
+
+	can_construct_potential = {
+		building_aviary_complex_requirement_terrain = yes
+		building_requirement_castle_city_church = { LEVEL = 01 }
+		has_unlocked_aerial_maa_trigger = yes
```

Comment:
Does it creates an error? It's culture triggered placed outside the culture scope.